### PR TITLE
Migrate examples to Goerli

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

File: _includes/samples/VRF/RandomNumberConsumer.sol [hunk 1]
```diff
@@ -18,24 +18,23 @@ contract RandomNumberConsumer is VRFConsumerBase {
...
     /**
      * Constructor inherits VRFConsumerBase
      *
-     * Network: Rinkeby
-     * Chainlink VRF Coordinator address: 0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B
-     * LINK token address: 0x01BE23585060835E02B77ef475b0Cc51aA1e0709
+     * Network: Goerli
+     * Chainlink VRF Coordinator address: 0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D
+     * LINK token address: 0x326C977E6efc84E512bB9C30f76E30c160eD06FB
      * Key Hash: 0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311
```
Comment (rgottleber) [inline _includes/samples/VRF/RandomNumberConsumer.sol:29]: Does this need to be 
`     * Key Hash: 0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15`

**Review by rgottleber — DISMISSED** [review]

**Comment by dwightjl** [comment]
Rebased to main

File: docs/Introduction/tutorials/intermediates-tutorial.md [hunk 2]
```diff
@@ -103,12 +103,12 @@ contract VRFD20 is VRFConsumerBaseV2 {
...
 ### Contract variables
 
-This example is adapted for [Rinkeby testnet](/docs/vrf-contracts/#rinkeby-testnet) but you can change the configuration and make it run for any [supported network](/docs/vrf-contracts/#configurations).
+This example is adapted for [Goerli testnet](/docs/vrf-contracts/#goerli-testnet) but you can change the configuration and make it run for any [supported network](/docs/vrf-contracts/#configurations).
 
 ```solidity
 uint64 s_subscriptionId;
-address vrfCoordinator = 0x6168499c0cFfCaCD319c818142124B7A15E857ab;
-bytes32 s_keyHash = 0xd89b2bf150e3b9e13446986e571fb9cab24b13cea0a43ea20a6049a85cc807cc;
+address vrfCoordinator = 0x2bce784e69d2Ff36c71edcB9F88358dB0DfB55b4;
```
Comment (aelmanaa) [inline docs/Introduction/tutorials/intermediates-tutorial.md:110]:
```suggestion
address vrfCoordinator = 0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D;
```

File: docs/Introduction/tutorials/intermediates-tutorial.md [hunk 3]
```diff
@@ -103,12 +103,12 @@ contract VRFD20 is VRFConsumerBaseV2 {
...
+bytes32 s_keyHash = 0x0476f9a745b61ea5c0ab224d3a6e4c99f0b02fce4da01143a4f70aa80ae76e8a;
```
Comment (aelmanaa) [inline docs/Introduction/tutorials/intermediates-tutorial.md:111]:
```suggestion
bytes32 s_keyHash = 0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15;
```

File: _includes/samples/VRF/RandomNumberConsumer.sol [hunk 4]
```diff
@@ -18,24 +18,23 @@ contract RandomNumberConsumer is VRFConsumerBase {
...
      * Constructor inherits VRFConsumerBase
      *
-     * Network: Rinkeby
-     * Chainlink VRF Coordinator address: 0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B
-     * LINK token address: 0x01BE23585060835E02B77ef475b0Cc51aA1e0709
-     * Key Hash: 0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311
+     * Network: Goerli
+     * Chainlink VRF Coordinator address: 0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D
+     * LINK token address: 0x326C977E6efc84E512bB9C30f76E30c160eD06FB
+     * Key Hash: 0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15
```
Comment (aelmanaa) [inline _includes/samples/VRF/RandomNumberConsumer.sol:29]:
```suggestion
     * Network: Goerli
     * Chainlink VRF Coordinator address: 0x2bce784e69d2Ff36c71edcB9F88358dB0DfB55b4
     * LINK token address: 0x326C977E6efc84E512bB9C30f76E30c160eD06FB
     * Key Hash: 0x0476f9a745b61ea5c0ab224d3a6e4c99f0b02fce4da01143a4f70aa80ae76e8a
```

File: _includes/samples/VRF/RandomNumberConsumer.sol [hunk 5]
```diff
@@ -18,24 +18,23 @@ contract RandomNumberConsumer is VRFConsumerBase {
...
      */
     constructor()
         VRFConsumerBase(
-            0xb3dCcb4Cf7a26f6cf6B120Cf5A73875B7BBc655B, // VRF Coordinator
-            0x01BE23585060835E02B77ef475b0Cc51aA1e0709  // LINK Token
+            0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D, // VRF Coordinator
```
Comment (aelmanaa) [inline _includes/samples/VRF/RandomNumberConsumer.sol:33]:
```suggestion
            0x2bce784e69d2Ff36c71edcB9F88358dB0DfB55b4, // VRF Coordinator
```

File: _includes/samples/VRF/RandomNumberConsumer.sol [hunk 6]
```diff
@@ -18,24 +18,23 @@ contract RandomNumberConsumer is VRFConsumerBase {
...
+            0x326C977E6efc84E512bB9C30f76E30c160eD06FB  // LINK Token
         )
     {
-        keyHash = 0x2ed0feb3e7fd2022120aa84fab1945545a9f2ffc9076fd6156fa96eaff4c1311;
+        keyHash = 0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15;
```
Comment (aelmanaa) [inline _includes/samples/VRF/RandomNumberConsumer.sol:37]:
```suggestion
        keyHash = 0x0476f9a745b61ea5c0ab224d3a6e4c99f0b02fce4da01143a4f70aa80ae76e8a;
```

**Review by aelmanaa — CHANGES_REQUESTED** [review]

File: docs/Developer Reference/acquire-link.md [hunk 7]
```diff
@@ -16,7 +16,7 @@ To see your LINK token balance in MetaMask, you must manually add the token.
 
 1. Open up MetaMask.
 1. At the bottom of the MetaMask windows, click **Import tokens**.
-1. Find the LINK token contract address for the network that you want to use. On Rinkeby our LINK token address is: `0x01BE23585060835E02B77ef475b0Cc51aA1e0709`. See the [LINK Token Contracts](/docs/link-token-contracts/) page to find addresses for different testnets.
+1. Find the LINK token contract address for the network that you want to use. On Goerli the LINK token address is: `0x326C977E6efc84E512bB9C30f76E30c160eD06FB`. See the [LINK Token Contracts](/docs/link-token-contracts/) page to find addresses for different testnets.
 1. Paste the token contract address into MetaMask in the Token Address input. The token symbol and decimals of precision will auto-populate.
     ![Metamask Custom Tokens Screen](/images/getting-started/metamaskImportTokens.png)
```
[the remark this answers is not in the document]
Comment (dwightjl) [reply docs/Developer Reference/acquire-link.md]: Screenshot updated.

**Review by aelmanaa — APPROVED** [review]

File: _includes/samples/PriceFeeds/PriceConverter.sol [hunk 8]
```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.7;
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 
 /**
- * Network: Rinkeby
+ * Network: Rinkeby (No Goerli Feed for EUR/USD at this time)
```
Comment (bryanjowers) [inline _includes/samples/PriceFeeds/PriceConverter.sol:7]: what impact will this have on our docs?

Comment (dwightjl) [reply _includes/samples/PriceFeeds/PriceConverter.sol:7]: The priceconverter example remains on Rinkeby. Alternatively we can point it to mainnet feeds but we want developers to be able to test it. If we can get some forex feeds on Goerli, that would help.

**Comment by dwightjl** [comment]
Rebased to main

**Pull request merged**